DbTables/inc/DIRACtoCalo.hh

```diff
@@ -0,0 +1,56 @@
+#ifndef DbTables_DIRACtoCalo_hh
+#define DbTables_DIRACtoCalo_hh
+
+
+#include <string>
+#include <iomanip>
+#include <sstream>
+#include <map>
+#include "DbTables/inc/DbTable.hh"
+
+namespace mu2e {
+
+  class DIRACtoCalo : public DbTable {
+  public:
+
+    typedef std::shared_ptr<DIRACtoCalo> ptr_t;
+    typedef std::shared_ptr<const DIRACtoCalo> cptr_t;
+    constexpr static const char* cxname = {"DIRACtoCalo"};
+    
+    class Row {
+    public:
+      Row(int index, uint16_t caloRoId):_index(index),_caloRoId(caloRoId) {}
+      int  index() const { return _index;}
+      uint16_t caloRoId() const {return _caloRoId;}
+    private:
+      int _index;
+      uint16_t _caloRoId;
+    };
+
+
+    DIRACtoCalo():DbTable(cxname,"calo.diractocalo",
+			  "index,caloRoId") {}
+    const Row& rowAt(const std::size_t index) const { return _rows.at(index);}
+    std::vector<Row> const& rows() const {return _rows;}
+    std::size_t nrow() const { return _rows.size(); };
+    virtual std::size_t nrowFix() const { return 216; }; 
```

Comment:
good point. We never had the chance to test it because it requires CaloFragment generated in a consisted way. done 
We are adding, as it was suggested before, a simple class named `DataProducts/inc/CaloId.hh` to house all the CaloDAQ numbers we need. This will also fix the circular dependence recently introduced in the calorimeter geoemtry.
(Giani &Stefano).